apis/operator/v1alpha1/types_cert_manager.go

```diff
@@ -5,11 +5,13 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+// CertManager provides information to configure an operator to manage certmanager.
+//+kubebuilder:object:root=true
```

Comment:
With the Makefile change above, changes to this file should not be necessary

Reply:
Done.